libs/langgraph/src/pregel/loop.ts

```diff
@@ -311,7 +315,13 @@ export class PregelLoop {
       // Start the store. This is a batch store, so it will run continuously
       store.start();
     }
-
+    if (checkSubgraphs && isNested && params.checkpointer !== undefined) {
+      if (_SEEN_CHECKPOINT_NS.has(config.configurable?.checkpoint_ns)) {
+        throw new MultipleSubgraphsError();
```

Comment:
Would be nice to have a better error message here (if we know the name where this is occurring, recommendation on what to do)